schematic/store/synapse.py

```diff
@@ -1517,6 +1517,60 @@ async def store_async_annotation(self, annotation_dict: dict) -> Annotations:
         )
         return await annotation_class.store_async(synapse_client=self.syn)
 
+    def process_row_annotations(
+        self,
+        dmge: DataModelGraphExplorer,
+        metadata_syn: Dict[str, Any],
+        hide_blanks: bool,
+        csv_list_regex: str,
+        annos: Dict[str, Any],
+        annotation_keys: str,
+    ) -> Dict[str, Any]:
+        """processes metadata annotations
```

Comment:
Could you update the docstring with a small bullet point list on what it means to "processes metadata annotations"?